src/components/price.component.tsx

```diff
@@ -0,0 +1,142 @@
+import { Button } from "@/components/ui/button.tsx";
+import list_tem from "@/assets/price-list-item.svg";
+
+const data = {
```

Comment:
Maybe let's add localization (i18n) and move all text/content outside of components?

Reply:
Will be done in separate PR